Add "Apply Asset Tag with InsightVM from Slack"

PhilippBehmer opened the pull request:
WWF Workflow "Apply Asset Tag with InsightVM from Slack"

jmcadams-r7 commented on workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/Apply_Asset_Tag_with_InsightVM_from_Slack.icon:
```diff
@@ -0,0 +1,3381 @@
+        "name": "Apply Asset Tag with InsightVM from Slack",
+        "type": "runnable",
+        "version": "",
+        "description": "",
```
Need WF description.

jmcadams-r7 commented:
This is a neat workflow overall...very slick.

sengleson-r7 commented:
@PhilippBehmer this is an awesome workflow!!! Above and beyond what I had in mind, I love how you can create a tag on the fly.
Joey makes a good point in it would be nice to give users a way to cancel the workflow, but I think we could post this as a v1 and improve it.
Can you add a Description in the workflow itself? Once that's in, I think we can merge! Thanks!

PhilippBehmer commented:
- Updated IVM to 4.8.1
- Updated TC to 1.6.1
- Replaced the "Extract Parameters" step with the new words attribute of the Slack plugin (Good to see my IDEA ticket in action 🎉 )
- Added description

@jmcadams-r7 I wanted to add a cancel button, but ICON only allows 5 buttons. 
If the user wants to cancel he can just do nothing and wait for the timeout (1min)
I did this with "length({{["Join Criticality"].[Responding User]}}) > 0" before, but I just found out that we can add a timeout path to the Slack step and switched to that one.

jschipp-r7 commented on workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/workflow.spec.yaml:
```diff
@@ -0,0 +1,33 @@
+products: ["insightconnect"]
+name: Apply_Asset_Tag_with_InsightVM_from_Slack
+title: "Apply Asset Tag with InsightVM from Slack"
+description: This workflow enables you to set asset tags in InsightVM from Slack.
```
```suggestion
description: This workflow enables you to set asset tags in InsightVM from Slack
```

PhilippBehmer commented:
❯ icon-validate ./Apply_Asset_Tag_with_InsightVM_from_Slack
[*] Validating workflow at ./Apply_Asset_Tag_with_InsightVM_from_Slack

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 27.893ms
